README_ru.md

```
@@ -0,0 +1,415 @@
+   lefthook install
+   ```
+
+3. Запустите зависимости (MinIO, Elasticsearch, Redis, MySQL):
+
+   ```bash
+   docker compose -f docker/docker-compose-base.yml up -d
+   ```
+
+   Добавьте в `/etc/hosts`:
+
+   ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)